### ISSUE TITLE
Is the project alive?

### ISSUE THREAD

**Issue by ghost** [rethinkdb/rethinkdb#6524]
Not much going on - what's the status?

**Comment by kofalt** [comment]
This might help: https://github.com/rethinkdb/rethinkdb/pulse/monthly

**Comment by notsonotso** [comment]
Not really

**Comment by Extarys** [comment]
Well, yes it helps - but yes it is alive. A quick search shows the project is now owned by the Linux Foundation now :)

And the database is fast and powerful :D

**Comment by srh** [comment]
It's not really that fast, but it's alive in the sense that it's being kept working on new platforms.

**Comment by notsonotso** [comment]
@srh thanks for the update!

**Comment by tavurth** [comment]
@notsonotso Running RethinkDB in a production system now. I chose this after diving deep into performance analytics and comparing benefits.

Query language is very powerful, consistency, atomicity, concurrency, sharding, all these normally headache inducing speed bumps are a no-brainer.

We're running at pre-release levels of traffic right now but I've never run into an issue with the RethinkDB driver itself, nor the query language.

Once you start nearing production, look at caching most of your common queries in Redis and you're good to go!

The thing I like most about using Rethink is the iterative building of queries. Today I wrote a complex abstract branching reduction of data, and after only one fix, it passes all tests. Not only that, I'm now able to reuse the fully test covered query for many different use cases.

From me, it gets production ready thumbs up!
